purchase_order_uninvoiced_amount/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Tecnativa - Manuel Calero
+# Copyright 2020 Tecnativa - João Marques
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Purchase Order Univoiced Amount",
```

Comment:
@SurachetS It would be great if you can add module summary.

Reply:
@MeetKD Updated :)